Description: Fix syntax error (missing comma) in AC_RUN_IFELSE.
   This bug causes arguments to be shifted. With the bug,
   presence of strcspn is misdetected everywhere and for cross
   compilation, the build simply fails.
Author: Helmut Grohne <helmut@subdivi.de>
Origin: vendor
Bug-Debian: https://bugs.debian.org/941025
Forwarded: no
Reviewed-By: Andreas Metzler <ametzler@debian.org>
Last-Update: 2019-10-05

--- config/ag_macros.m4
+++ config/ag_macros.m4
@@ -428,7 +428,7 @@ int main (int argc, char ** argv) {
    char zRej@<:@@:>@ = reject;
    char zAcc@<:@@:>@ = "a-ok-eject";
    return strcspn( zAcc, zRej ) - 5;
-}] )]
+}] )],
     [ag_cv_run_strcspn=yes],[ag_cv_run_strcspn=no],[ag_cv_run_strcspn=no]
   ) # end of RUN_IFELSE
   ]) # end of AC_CACHE_VAL for ag_cv_run_strcspn
